plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator_test.go

```diff
@@ -458,3 +459,92 @@ valuesMerge: replace
 			111,             // memory
 		))
 }
+
+func TestHelmChartInflationGeneratorWithIncludeCRDs(t *testing.T) {
+	th := kusttest_test.MakeEnhancedHarnessWithTmpRoot(t).
+		PrepBuiltin("HelmChartInflationGenerator")
+	defer th.Reset()
+	if err := th.ErrIfNoHelm(); err != nil {
+		t.Skip("skipping: " + err.Error())
+	}
+
+	// we store this data outside of the _test.go file as its sort of huge
+	// and has backticks, which makes string literals wonky
+	testData, err := ioutil.ReadFile("include_crds_testdata.txt")
```

Comment:
Related to the large text file that I added.

`CustomResourceDefinitions` are big, and I felt it was best to not include it as a string literal inside this test file.

Let me know if this isn't the best way to do this, and I'll be glad to put it back into this file.

Reply:
shouldn't be a problem.